src/cl/lighthouse/lighthouse_launcher.star

```diff
@@ -166,12 +166,14 @@ def launch(
 	beacon_metrics_port = beacon_service.ports[BEACON_METRICS_PORT_ID]
 	beacon_metrics_url = "{0}:{1}".format(beacon_service.ip_address, beacon_metrics_port.number)
 
-	validator_metrics_port = validator_service.ports[VALIDATOR_METRICS_PORT_ID]
-	validator_metrics_url = "{0}:{1}".format(validator_service.ip_address, validator_metrics_port.number)
+	if node_keystore_files != None:
+		validator_metrics_port = validator_service.ports[VALIDATOR_METRICS_PORT_ID]
+		validator_metrics_url = "{0}:{1}".format(validator_service.ip_address, validator_metrics_port.number)
+		validator_node_metrics_info = cl_node_metrics.new_cl_node_metrics_info(validator_node_service_name, METRICS_PATH, validator_metrics_url)
 
 	beacon_node_metrics_info = cl_node_metrics.new_cl_node_metrics_info(beacon_node_service_name, METRICS_PATH, beacon_metrics_url)
-	validator_node_metrics_info = cl_node_metrics.new_cl_node_metrics_info(validator_node_service_name, METRICS_PATH, validator_metrics_url)
-	nodes_metrics_info = [beacon_node_metrics_info, validator_node_metrics_info]
+
+	nodes_metrics_info = [beacon_node_metrics_info, validator_node_metrics_info if node_keystore_files != None else None]
```

Comment:
how does the `None` get interpreted down stream in the config yml; can we do something like

```py
nodes_metrics_info = [beacon_node_metrics_info]
if validator_node_metrics_info:
     nodes_metrics_info.append(validator_node_metrics_info)
```